### ISSUE TITLE
[BUG] Can't complete SAP SSO Authentication

### ISSUE THREAD

**Issue by jasontenpenny** [Azure/sap-automation#460]
**Describe the bug**
I haven't been able to use the automation script to download SAP software. Every time I try to run the script, it fails to complete the SAP SSO/SAML authentication. It always fails on step 5 with the same error message.

I've discussed the issue with others on my team, and they haven't seen it. However, a potential difference is that I have gone through the process of setting up an SAP Universal ID, and most of the others on my team haven't done that yet. Possibly that is changing the authentication flow. 

Without knowing much about SAML, my theory is that the post_action generated by the Universal ID is now a complete URL that doesn't need the origin URL in front anymore. I've noticed in the output that the post_url appears to be invalid, as seen in the screenshot below. 
![image](https://github.com/Azure/sap-automation/assets/50330957/54c56dd0-27ba-4a5f-a466-d3529b49c02b)

I've tried to run the install using both my S-user ID as username as well as the email address associated with the account. I've tried reinstalling all the components on my VM, nothing has allowed me to proceed all the way through the authentication.

**To reproduce**
I'm unclear on exact steps to reproduce. My theory is to set up an SAP Universal ID and link it to your S-user account. Then attempt to download SAP software using the script.

**Expected behavior**
I'd expect the SSO authentication to complete so I can actually download files from SAP.

**Additional context**
I don't think there's a way for me to remove the SAP Universal ID, and that supposedly will become required soon. https://www.sap.com/account/universal-id.html

**Comment by hdamecharla** [comment]
Hi @mkdeegan , could you kindly take a look at this please?

**Comment by jkirkmannz** [comment]
It should be the S-UserID rather than the email address used for the SAP Universal ID. Try resetting the password associated with the S-UserID, as this can expire but still allow access to the SAP site via the Universal ID and selecting the S-UserID.

**Comment by jasontenpenny** [comment]
@jkirkmannz I have tried to use both the S-User ID as well as the email address associated with the account. I have reset the password in two different places, which I think would cover the S-user pwd as well as the universal ID password. I set both to the same, but that still doesn't let it work.

**Comment by jasontenpenny** [comment]
It looks like I was able to resolve this. Apparently there is a password reset tool at both account.sap.com as well as accounts.sap.com. It appears that account.sap.com is "legacy" and tied directly to the S-user ID, whereas accounts.sap.com is the Universal ID.

Apparently both password resets I had done were on the Universal ID side of things. Today I found the reset password option at account.sap.com, and I am now completing the authentication and able to download files.
<img width="231" alt="image" src="https://github.com/Azure/sap-automation/assets/50330957/8c84d5e0-169b-4766-b33a-c6830d38e2fc">
